packages/tests/src.ts/test-wallet.spec.ts

```diff
@@ -650,4 +651,59 @@ describe("Wallet createAccount", function () {
 
         assert.strictEqual(exceptionThrown, true);
     }).timeout(timeout);
+
+    it("Should make a contract call with isCryptoTransfer false and set 'to' and 'value'", async function () {
+        let exceptionThrown = false;
+        try {
+            await acc1Wallet.sendTransaction({
+                to: acc2Wallet.account,
+                value: 1,
+                isCryptoTransfer: false,
+                gasLimit: 300000
+            });
+        } catch (e: any) {
+            exceptionThrown = true;
+            assert.strictEqual(e.code, 'INVALID_CONTRACT_ID');
+        }
+
+        assert.strictEqual(exceptionThrown, true);
+    }).timeout(timeout);
+
+    it("Should make a contract call with 'to' and 'value' with provided contract address as 'to'", async function() {
+        const abiGLDTokenWithConstructorArgs = JSON.parse(readFileSync('examples/assets/abi/GLDTokenWithConstructorArgs_abi.json').toString());
+        const contractByteCodeGLDTokenWithConstructorArgs = readFileSync('examples/assets/bytecode/GLDTokenWithConstructorArgs.bin').toString();
+        const contractFactory = new ethers.ContractFactory(abiGLDTokenWithConstructorArgs, contractByteCodeGLDTokenWithConstructorArgs, acc1Wallet);
+        const contract = await contractFactory.deploy(ethers.BigNumber.from('10000'), {gasLimit: 3000000});
+        await contract.deployed();
+
+        let exceptionThrown = false;
+        try {
+            await acc1Wallet.sendTransaction({
```

Comment:
This test should not fail. It should succeed. The reason that it does not succeed is that you have to specify the `gasLimit` as part of the TX. The caveat is that if you specify `gasLimit` it will not check the provider for the type of the `to` address (which is okay IMO)